.github/workflows/deploy.yml

```diff
@@ -69,23 +70,15 @@ jobs:
       - name: Copy built schema to SCHEMA_DIR
         run: cp -r dist/schemata/dev $SCHEMA_DIR/
 
-      - name: Check git status before commit
```

Comment:
I know that these three steps are somewhat verbose, but it helps to keep things separated (also for debugging). So would prefer not to combine them.

Reply:
Thank you